external_api/api.py

```diff
@@ -238,10 +240,15 @@ def entity_data_view(request, entity_name):
     """
     if request.method == 'GET':
         params = request.GET.dict()
-        # Fetch Languages supported by the entity
+
+        shuffle = (params.get('shuffle', 'false') or '').lower() == 'true'
```

Comment:
defensive coding
for example you can have 

```python
{}
{'shuffle': 'true'}
{'shuffle': 'false'}
{'shuffle': None}
```

In the last case since `shuffle` key exists `params.get('shuffle', 'false')` will return `None`, and since we want to deal with strings we do `or ''` i.e. `None or ''` which evals to `''`